src/Notes/MerchantEmailNotifications/MerchantEmailNotifications.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Handles merchant email notifications
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes\MerchantEmailNotifications;
+
+use Automattic\WooCommerce\Admin\Notes\Note;
+use Automattic\WooCommerce\Admin\Notes\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Merchant email notifications.
+ * This gets all non-sent notes type `email` and sends them.
+ */
+class MerchantEmailNotifications {
+	/**
+	 * Send all the notifications type `email`.
+	 */
+	public static function possible_send() {
+		$data_store = \WC_Data_Store::load( 'admin-note' );
+		$notes      = $data_store->get_notes(
+			array(
+				'type'   => array( Note::E_WC_ADMIN_NOTE_EMAIL ),
+				'status' => array( 'unactioned' ),
+			)
+		);
+
+		if ( empty( $notes ) ) {
```

Comment:
This isn't really needed as the `foreach` will handle an empty array, having this makes it slightly harder to grok imo.

Reply:
Good catch! Modified in the commit be9301d